desmume/src/frontend/posix/shared/mic_openal.cpp

```diff
@@ -1,25 +1,30 @@
 /*
-	Copyright (C) 2009-2010 DeSmuME team
-
-	This file is free software: you can redistribute it and/or modify
-	it under the terms of the GNU General Public License as published by
-	the Free Software Foundation, either version 2 of the License, or
-	(at your option) any later version.
-
-	This file is distributed in the hope that it will be useful,
-	but WITHOUT ANY WARRANTY; without even the implied warranty of
-	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-	GNU General Public License for more details.
-
-	You should have received a copy of the GNU General Public License
-	along with the this software.  If not, see <http://www.gnu.org/licenses/>.
+	Copyright (C) 2009-2010 DeSmuME team
```

Comment:
it appears your editor somehow produced gratuitous whitespace changes (if you use `git add --patch filename` you could exclude the hunk)

Reply:
Turns out this file is using Windows line endings in this comment and Unix line endings everywhere else. Would it be alright to commit it like this to fix the inconsistency? If not, I can try to configure Git to commit to commit the line endings without changing them, but a simple `git add --patch` doesn't help here.

Reply:
> a simple git add --patch doesn't help here.

yeah, you'd need to unstage the commit beforehand and commit again.

> Turns out this file is using Windows line endings in this comment and Unix line endings everywhere else.

yeah' there's a few such inconsistencies in the codebase, my editor also "fixed" a couple unrelated lines when editing some files, though i noticed before commiting by looking at the changes with `gitk`, so i used git add --patch to exclude those hunks. but yeah, i'm personally ok with having the whitespace fixed in this case, it would be nicer if it was a separate commit but let's not be too bureaucratic. i don't think @zeromus cares either.

Reply:
Alright, I've restored the original file endings. FYI in case anyone else has this issue in the future, unstaging and commiting again didn't work until I've disabled auto CRLF with `git config --global core.autocrlf false`.